knowledge-base/chart-dash-series.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Chart: Dashed line series
+description: Modify the line chart to have dashed lines by using the native chart.
```

Comment:
remove native chart from this description:
Create dashed line chart series using line series on Android, iOS, MacCatalyst, WinUI